jobs/rocoto/metp.sh

```diff
@@ -13,14 +13,6 @@
 ## METPCASE : METplus verification use case (g2g1 | g2o1 | pcp1)
 ###############################################################
 
-###############################################################
-echo
-echo "=============== START TO SOURCE FV3GFS WORKFLOW MODULES ==============="
-. $HOMEgfs/ush/load_fv3gfs_modules.sh
```

Comment:
This might be non-consequential to remove, but good to verify on a R&D machine.

Reply:
I know without even trying that MET+ uses modules. There are specific modules just for that in module_base.

Reply:
The load_fv3gfs_modules.sh conversation is happened in above. It is a repeat conversation here. We can continue this topic in the original message dialog above. Will mark as resolved here.